tests/topotests/static_routing_mpls/test_static_routing_mpls.py

```diff
@@ -134,6 +152,89 @@ def test_mpls_configured_on_interface():
     assert _check_mpls_state(tgen.gears["r2"], "r2-eth2", False), assertmsg
 
 
+def test_mpls_zebra_route_nexthop():
+    "Test 'mpls' state is correctly configured with labeled routes configured"
+
+    tgen = get_topogen()
+    # Don't run this test if we have any failure.
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    router = tgen.gears["r2"]
+    output = router.vtysh_cmd("config terminal\nmpls lsp 33 192.168.2.3 implicit-null")
```

Comment:
you don't check/print the output of `output`, we can just remove this assignment...

Reply:
fixed